filters/diag/diag.go

```diff
@@ -417,3 +457,103 @@ func (t *throttle) Response(ctx filters.FilterContext) {
 	rsp := ctx.Response()
 	rsp.Body = t.goThrottle(rsp.Body, true)
 }
+
+func (j *jitter) Name() string {
+	switch j.typ {
+	case normalDist:
+		return "normal"
+	case gaussianDist:
+		return "gaussian"
+	case randomDist:
+		return "random"
+	}
+	return "unknown"
+}
+
+func (j *jitter) CreateFilter(args []interface{}) (filters.Filter, error) {
+	var (
+		delay     time.Duration
+		interval  time.Duration
+		mu, sigma float64
+		err       error
+	)
+
+	if !(len(args) == 2 || len(args) == 4) {
+		return nil, errors.New("filter requires 2 or 4 parameters")
+	}
+
+	if delay, err = parseDuration(args[0]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration delay %v: %w", args[0], err)
+	}
+
+	if interval, err = parseDuration(args[1]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration interval %v: %w", args[1], err)
+	}
+
+	switch j.typ {
+	case normalDist:
+		mu = 0
+		sigma = 1
+
+	case gaussianDist:
+		if len(args) != 4 {
+			return nil, errors.New("gaussian jitter filter requires 4 parameters")
+		}
+
+		s, ok := args[2].(string)
+		if !ok {
+			return nil, fmt.Errorf("failed to convert mu to string %v", args[2])
+		}
+
+		mu, err = strconv.ParseFloat(s, 32)
```

Comment:
I think we dont need to parse and can just check it is float64 like we do for numbers elsewhere (in any case `ParseFloat` should use 64 bits)

Reply:
Yes, I just need to convert to float, no need to go through string :)
I am just writing tests, right now. If you have naming